src/test/regress/expected/alter_distributed_table.out

```diff
@@ -837,5 +837,35 @@ SELECT * FROM mat_view ORDER BY a;
  2 | 2
 (2 rows)
 
+-- test long table names
+SET client_min_messages TO DEBUG1;
```

Comment:
given that we touch partitioned tables in the code, can you add tests for it as well? Also, for simple RENAME command as well

Reply:
done and done